index.html

```diff
@@ -0,0 +1,49 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
```

Comment:
- [ ] Please kindly address this spacing issue. 👍 I observed that few elements are outside the parent container. Thanks. 🙏 

## Your Design

![png-transparent-nigerian-naira-currency-symbol-symbol-miscellaneous-angle-text](https://user-images.githubusercontent.com/103386183/212033698-5bd33fe4-2942-4900-87c7-c42fe2d29975.png)

Reply:
Hi @schoolsofafrika, try to regroup your elements` parent container and play around with your CSS file.

**Figma template**
![Screen Shot 2023-01-12 at 15 47 19](https://user-images.githubusercontent.com/98527559/212047388-09c0c376-b206-42a1-a030-085121534ee3.png)

**Yours**
![Screen Shot 2023-01-12 at 15 47 39](https://user-images.githubusercontent.com/98527559/212047410-ce729a6f-3d57-424d-b4ae-d720fad24ff6.png)